swift_browser_ui_frontend/src/views/Upload.vue

```diff
@@ -1,289 +1,109 @@
 <template>
```

Comment:
`swift_browser_ui_frontend/src/views/Upload.vue` should be moved to `swift_browser_ui_frontend/src/components/Upload.vue`, and probably renamed to `UploadModal.vue` to adhere to vue naming convention. `views` folder is reserved for routing root views, and upload functions as a component now. This should've probably been documented, I'll make a PR out of it afterwards.